src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -448,6 +460,17 @@ def shim_to_feedback(
     legacy user report and event to create the new feedback.
     """
     if is_in_feedback_denylist(project.organization):
+        track_outcome(
+            org_id=project.organization_id,
+            project_id=project.id,
+            key_id=None,
+            outcome=Outcome.ABUSE,
+            reason=None,
```

Comment:
This outcome type is currently only emitted by nginx, so I would tend to use `Outcome.RATE_LIMITED` here with a specific `reason` here (e.g. `reason="feedback_denylist").